pulp_container/app/models.py

```diff
@@ -470,6 +547,10 @@ class ContainerRepository(
     PUSH_ENABLED = False
     ACCESS_POLICY_VIEWSET_NAME = "repositories/container/container"
 
+    manifest_signing_service = models.ForeignKey(
+        ManifestSigningService, on_delete=models.SET_NULL, null=True
```

Comment:
Should we protect the signing service to be deleted while in use? Not sure about the business logic here.

Reply:
That's a good question - but this question stands for remote usage as well. If there are 2 repos that are associated with same remote that is deleted, both repos will loose it

Reply:
let's talk about this on pulpcore meeting